doc/source/whatsnew/v0.15.0.rst

```diff
@@ -7,10 +7,11 @@ v0.15.0 (October 18, 2014)
 
 .. ipython:: python
    :suppress:
-
+   
    from pandas import * # noqa F401, F403
```

Comment:
we want to delete this whole block, that's one of the main points of the issue (see the description)

Reply:
Just to be sure, you want me to remove the whole ipython block, am i right?

Reply:
yes, you can see a similar case here: https://github.com/pandas-dev/pandas/pull/24236/files